model/gurps/skill.go

```diff
@@ -916,11 +921,11 @@ func (s *Skill) bestDefault(excluded *SkillDefault) *SkillDefault {
 
 func (s *Skill) calcSkillDefaultLevel(def *SkillDefault, excludes map[string]bool) fxp.Int {
 	e := EntityFromNode(s)
-	level := def.SkillLevel(e, s.Replacements, true, excludes, !s.IsTechnique())
-	if def.SkillBased() {
+	level := def.SkillLevel(e, s.Replacements, !e.SheetSettings.UseSkillTrees, excludes, !s.IsTechnique())
+	if def.SkillBased() && e != nil && e.SheetSettings.UseSkillTrees {
```

Comment:
The old code did not skip this computation if there was no entity, so this seems wrong again.